src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -35,8 +37,8 @@ public ErrorLogger(Stream stream, string toolName, string toolFileVersion, Versi
             _writer.WriteObjectStart(); // root
             _writer.Write("version", OutputFormatVersion);
 
```

Comment:
I can emit this, but I'll wait until that URL does not have anything beta in it and it actually has the final schema. If we ship U3 against a spec still in beta then beta -> RTM changes will cause U3 logs to fail to validate. I know the plan is not for that to happen, but I'll be conservative and add $schema only when we change 'version' to '1.0.0' in the output.